### PR TITLE
[refactor] Move db connection out services

### PR DESCRIPTION
**Pull request by Rostislav013**
So I have moved db connection to src. 
There are some warns in that db connection file with any types. Will have a look at it. But doesn't affect merging.  
Db connection is taken from `https://node-postgres.com/guides/project-structure`.

### CODE REVIEW

File: src/services/user.ts [hunk 1]
```diff
@@ -91,15 +79,13 @@ const updateUser = async (userId: string, update: Partial<User>) => {
...
   const user: User = DBResponse.rows[0]
 
   if (!user) {
     throw new Error()
   } else {
-    pool.query('DELETE FROM userk WHERE user_id = $1;', [userId], (err) => {
-      if (err) throw err
-    })
+    await db.query('DELETE FROM userk WHERE user_id = $1;', [userId])
     return { message: 'User deleted' }
```
Comment (MikeVedsted) [inline src/services/user.ts:89]: how do we handle potential errors in cases like this? Could we maybe add `...rest` to the props of your query function?

File: src/db/index.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+})
+
+export default {
+  async query(text: string, params?: any[]) {
+    const start = Date.now()
+    const res = await pool.query(text, params)
+    const duration = Date.now() - start
+    console.log('executed query', { text, duration, rows: res.rowCount })
+    return res
+  },
```
Comment (MikeVedsted) [inline src/db/index.ts:22]: will it work to add `...rest` here for when we need to add functions?

**Review by MikeVedsted — APPROVED** [review]
nice work

**Review by Chiranjibichapagain — APPROVED** [review]

File: src/services/user.ts (hunk 1 above)
Comment (Rostislav013) [reply src/services/user.ts:89]: Cuz we use async await here, it is not needed. Controller catches it

**Pull request merged**